src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Plane myPlane = new Plane.PlaneBuilder()
+                .withColour("blue")
+                .withId(543)
+                .withOwner("Marsel")
+                .withNumberOfFlights(165)
+                .withModel("ad123")
+                .build();
```

Comment:
let's not push main class to github. We have tests to test this solution. (At the same time you can use it locally to test yourself).